examples/post_training_quantization/torch/mobilenet_v2/main.py

```diff
@@ -0,0 +1,189 @@
+"""
+ Copyright (c) 2022 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import os
+import re
+import subprocess
+import tempfile
+from pathlib import Path
+from typing import Tuple
+
+import nncf
+import numpy as np
+import openvino.runtime as ov
+import torch
+from fastdownload import FastDownload
+from openvino.tools import mo
+from sklearn.metrics import accuracy_score
+from torchvision import datasets, models, transforms
+from tqdm import tqdm
+
+ROOT = Path(__file__).parent.resolve()
+CHECKPOINT_URL = 'https://huggingface.co/alexsu52/mobilenet_v2_imagenette/resolve/main/pytorch_model.bin'
+DATASET_URL = 'https://s3.amazonaws.com/fast-ai-imageclas/imagenette2-320.tgz'
+DATASET_PATH = '~/.nncf'
```

Comment:
`~/.cache/nncf/datasets/` should be more appropriate, IMO. At least, `~/.cache/...` is the folder that is used by default by may other libraries, including torch.

Reply:
Done